llm_on_ray/inference/predictor_deployment.py

```diff
@@ -58,16 +58,16 @@ def __init__(self, infer_conf: InferenceConfig):
         self.use_vllm = infer_conf.vllm.enabled
 
         if self.use_deepspeed:
-            from deepspeed_predictor import DeepSpeedPredictor
+            from llm_on_ray.inference.deepspeed_predictor import DeepSpeedPredictor
 
             self.predictor = DeepSpeedPredictor(infer_conf)
             self.streamer = self.predictor.get_streamer()
         elif self.use_vllm:
-            from vllm_predictor import VllmPredictor
+            from llm_on_ray.inference.vllm_predictor import VllmPredictor
 
             self.predictor = VllmPredictor(infer_conf)
         else:
-            from transformer_predictor import TransformerPredictor
+            from llm_on_ray.inference.transformer_predictor import TransformerPredictor
```

Comment:
try to reduce by import the module by adding TransformerPredictor in `__all__`
```suggestion
            from llm_on_ray.inference import TransformerPredictor
```